### PR TITLE
server: use debug level for connection error.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: src/server/server.rs [hunk 1]
```diff
@@ -308,29 +308,29 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
...
-                warn!("missing conn for token {:?}", token);
+                debug!("missing conn for token {:?}", token);
                 return;
             }
             Some(conn) => conn.append_write_buf(event_loop, data),
         };
 
         if let Err(e) = res {
-            warn!("handle write data err {:?}, remove", e);
+            debug!("handle write data err {:?}, remove", e);
```
Comment (BusyJay) [inline src/server/server.rs]: Why debug here?

Comment (siddontang) [inline src/server/server.rs]: it is no need to log network error with warning level.

Comment (BusyJay) [inline src/server/server.rs]: Why L158 use warn then?

Comment (siddontang) [inline src/server/server.rs]: what I said before that the store connection can't be closed arbitrarily, so only use warn here to tell us that store connection is removed.

**Comment by disksing** [comment]
LGTM

**Pull request merged**